WindowsServerDocs/identity/securing-privileged-access/securing-privileged-access.md

```diff
@@ -56,11 +56,11 @@ The roadmap is designed to maximize the use of Microsoft technologies that you a
 
 The roadmap of Microsoft recommendations is broken into 3 phases:
 
-* [Phase 1: First 30 days]()
+* [Phase 1: First 30 days](https://docs.microsoft.com/en-us/windows-server/identity/securing-privileged-access/securing-privileged-access#phase-1-quick-wins-with-minimal-operational-complexity)
```

Comment:
Please remove the en-us reference in each of these links to allow localized content to work properly. Thanks